libs/feature/question-types/src/lib/question-type-registry.tsx

```diff
@@ -114,7 +114,20 @@ export const EVALUATION_FUNCTIONS: { [QType in QuestionType["type"]]: Evaluation
 export const INITIAL_ANSWER_VALUE_FUNCTIONS: {
 	[QType in QuestionType["type"]]: InitialAnswerFn<QType>;
 } = {
-	"multiple-choice": () => ({}),
+	"multiple-choice": question => {
+		const answer: MultipleChoice["answer"]["value"] = {};
+
+		// Check if randomization is enabled
+		const answersToProcess = question.randomizeAnswers
+			? question.answers.sort(() => Math.random() - 0.5) 
+			: question.answers; 
+
+		for (const answerId of answersToProcess.map(a => a.answerId)) {
+			answer[answerId] = false;
+		}
```

Comment:
wenn eh eine for-schleifen verwendet wird, sollte hier auf den zusätzlich map-loop verzichtet werden. 

```suggestion
for (const { answerId } of answersToProcess) {
   answer[answerId] = false
}
```